get-image-version/index.js

```diff
@@ -135,7 +135,7 @@ function tryGetInfoFromDirectoryBuildProps() {
 function pushOutputs(branchName, prefix, suffix, moduleId) {
     branchName = branchName.substring(branchName.lastIndexOf('/') + 1, branchName.length).toLowerCase();
     const sha = github.context.eventName.startsWith('pull_request') ? github.context.payload.pull_request.head.sha.substring(0, 8) : github.context.sha.substring(0, 8);
-    const fullSuffix = suffix + '-' + branchName;
+    const fullSuffix = (suffix.length > 0) ? suffix + '-' + branchName : branchName;
```

Comment:
I suppose we can use: suffix ? suffix + '-' + branchName : branchName;
In js an empty string is always false in boolean

Reply:
Fixed